### PR TITLE
Feat - add option to make payment block fluid

### PR DESCRIPTION
**Pull request by nico3333fr**
In order to achieve design for VPN signup, I had to add a property fluidDisplay on payment component, if it is "activated", it adds some classes on the field container.

image

if this is not activated, we have standard behaviour:
image

### CODE REVIEW

File: containers/payments/Payment.js [hunk 1]
```diff
@@ -22,7 +23,8 @@ const Payment = ({
     method,
     onMethod,
     onValidCard,
-    onPay
+    onPay,
+    fluidDisplay
```
Comment (EpokK) [inline containers/payments/Payment.js]: I prefer if you define `fieldClassName` prop (`String`).

File: containers/payments/Payment.js [hunk 2]
```diff
@@ -70,7 +72,7 @@ const Payment = ({
         <>
             <Row>
                 <Label>{c('Label').t`Payment method`}</Label>
-                <Field>
+                <Field className={classnames([fluidDisplay && 'auto flex-item-fluid-auto'])}>
```
Comment (EpokK) [inline containers/payments/Payment.js]: And use it here:
```suggestion
                <Field className={fieldClassName}>
```

File: containers/payments/Payment.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
+import { classnames } from '../../helpers/component';
```
Comment (EpokK) [inline containers/payments/Payment.js]: And you don't need this line.

File: containers/payments/Payment.js [hunk 4]
```diff
@@ -111,7 +113,8 @@ Payment.propTypes = {
     onMethod: PropTypes.func,
     onValidCard: PropTypes.func,
     cycle: PropTypes.oneOf([CYCLE.MONTHLY, CYCLE.YEARLY, CYCLE.TWO_YEARS]),
-    onPay: PropTypes.func
+    onPay: PropTypes.func,
+    fluidDisplay: PropTypes.bool
```
Comment (EpokK) [inline containers/payments/Payment.js]:
```suggestion
    fieldClassName: PropTypes.string
```

File: containers/payments/Payment.js (hunk 3 above)
Comment (nico3333fr) [reply containers/payments/Payment.js]: Done: https://github.com/ProtonMail/react-components/pull/266/commits/b8eb830b7129fc6dfc70a4bad7ecc31d9e4c2bb3

File: containers/payments/Payment.js (hunk 1 above)
Comment (nico3333fr) [reply containers/payments/Payment.js]: Okay, done: https://github.com/ProtonMail/react-components/pull/266/commits/b8eb830b7129fc6dfc70a4bad7ecc31d9e4c2bb3

**Pull request merged**